ivy_tests/test_ivy/test_functional/test_extensions/test_core/test_general.py

```diff
@@ -0,0 +1,48 @@
+# global
+from hypothesis import given, strategies as st
+
+# local
+import numpy as np
+import ivy_tests.test_ivy.helpers as helpers
+from ivy_tests.test_ivy.helpers import handle_cmd_line_args
+
+
+# isin
+@handle_cmd_line_args
+@given(
+    dtype_and_x=helpers.dtype_and_values(available_dtypes=helpers.get_dtypes("numeric"),
+                                         num_arrays=2,
+                                         shared_dtype=True),
+    num_positional_args=helpers.num_positional_args(fn_name="isin"),
+    invert=st.booleans(),
+    assume_unique=st.booleans()
+)
+def test_isin(
+    dtype_and_x,
+    invert,
+    assume_unique,
+    as_variable,
+    num_positional_args,
+    native_array,
+    container,
+    instance_method,
+    fw,
+):
+    dtypes, values = dtype_and_x
+    elements, test_elements = values
+    helpers.test_function(
+        input_dtypes=dtypes,
+        as_variable_flags=as_variable,
+        with_out=False,
+        num_positional_args=num_positional_args,
+        native_array_flags=native_array,
+        instance_method=instance_method,
+        container_flags=container,
+        fw=fw,
+        fn_name="isin",
+        ground_truth_backend='numpy',
+        elements=elements,
+        test_elements=test_elements,
+        invert=invert,
+        assume_unique=assume_unique,
```

Comment:
The `assume_unique` parameter seems to produce unpredictable/incorrect behavior when set to `True` when `fw` is `'tensorflow'`. 😅

Reply:
> Hi @pillarxyz , Great work so far, and apologies once again for the late review 😅 I have ran the test that you have implemented and found that it still throws some errors. I've already fixed most of the trivial ones which were causing some of the backends to fail, and you can see the changes I've made in the commits.
> 
> Unfortunately however there still seems to be a problem with the behavior of the parameter `assume_unique` when it comes to the `tensorflow` backend implementation. Could you please try fixing that? and if you face any trouble you can ask me and I will hopefully reply faster this time for sure! 😅
> 
> Thanks!

One more note, as this bug is kind of flaky and not very easy to reproduce, if you are unable to reproduce it we can just merge the PR and handle it later if it starts failing :)